docs/en/pid-issuing.rst

```diff
@@ -0,0 +1,511 @@
+
+    PID Issuing - Detailed flow
+
+**Steps 1-4:** The User has selected a PID Provider, and the Wallet Instance obtains the metadata for the selected PID Provider.
+
+.. note::
+
+    **Federation Check:** The Wallet Instance needs to check if the PID Provider is part of Federation and then it can consume its Metadata.
+
+**Steps 5-6:** The Wallet Instance creates a fresh PKCE code verifier and sends this along in a *pushed authorization request* using the OIDC request parameter (see :rfc:`9126` Section 3) to the pushed authorization endpoint of the PID Provider. The Wallet Instance signs this request using its attested private key. We assume that a standard OAuth2 client authentication method should be involved as the pushed authorization endpoint is a protected endpoint. The client authentication can be based on the model defined in [:rfc:`7521`] using the Wallet Instance Attestation JWS inside the client_assertion parameter. The authorization_details [RAR :rfc:`9396`] parameter is extended to allow Wallet Instance to specify types of the credentials when requesting authorization for PID issuance.
+
+The following is a non-normative example of the PAR.
+
+.. code-block::
+
+    POST /as/par HTTP/1.1
+    Host: pid.it
+    Content-Type: application/x-www-form-urlencoded
+    
+    response_type=code
+    &client_id=$thumprint-of-the-jwk-in-the-cnf-wallet-attestation$
+    &code_challenge=E9Melhoa2OwvFrEMTJguCHaoeK1t8URWbuGJSstw-cM
+    &code_challenge_method=S256
+    &request=eyJhbGciOiJSUzI1NiIsImtpZCI6ImsyYmRjIn0.ew0KIC Jpc3MiOiAiczZCaGRSa3F0MyIsDQogImF1ZCI6ICJodHRwczovL3NlcnZlci5leGFtcGxlLmNvbSIsDQo gInJlc3BvbnNlX3R5cGUiOiAiY29kZSBpZF90b2tlbiIsDQogImNsaWVudF9pZCI6ICJzNkJoZFJrcXQz IiwNCiAicmVkaXJlY3RfdXJpIjogImh0dHBzOi8vY2xpZW50LmV4YW1...
+    &client_assertion_type=urn:ietf:params:oauth:client-assertion-type:jwt-bearer
+    &client_assertion=$WalletInstanceAttestation$ 
+
+The JWT payload of PAR is given as the following:
+
+.. code-block:: JSON
+
```

Comment:
please also add the jwt header here in a separate code-block

Reply:
@fmarino-ipzs I'm worried that `.. code-block:: JSON` compilation fails if found a dot between two json objects

Reply:
Done